src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,91 @@
+import React from 'react';
+import cn from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+
+  const countButtons = Math.ceil(total / perPage);
+  const buttons: number[] = Array.from(
+    { length: countButtons },
+    (_, i) => i + 1,
+  );
+  const isActive = (pageNumber: number): boolean => {
+    return pageNumber === currentPage;
+  };
+
+  const disableNext = currentPage === 1;
+  const disablePrev = currentPage === buttons.length;
+
+  const returnPage = () => {
+    if (disableNext) {
```

Comment:
The function `returnPage` should decrease the `currentPage` by 1 to go to the previous page. Ensure that the logic correctly handles the page change.